network/views.py

```diff
@@ -55,23 +62,43 @@ def post_detail(request, pk):
     #TODO: implement LIKE 
 
 
-@login_required
-def following(request):
-    # Query for requested data
+def follow_list(request, username, fol_type):
```

Comment:
can you put a little bit of description for some of these, the names dont feel descriptive enough to know what they are for.

Reply:
Okay, I added some comment